reports/Oregon.md

```diff
@@ -368,17 +381,51 @@ id: or-portland-19
 * [Tweet on June 17th enclosed FIA Request by Portland Attorney](https://twitter.com/alankesslr/status/1273299640310325250)
 
 
-### Police disperse press, tell them to run | June 16th
+### Protestor bleeding from head after he was shot by police | June 19th
 
-First-person footage from a protest on June 16th shows police telling press to leave the area. As press leave, an officer then says that if press doesn't move faster, they will go to jail. The filmer says, "So you want us to run?" The officer replies, "Yes."
+A screenshot of a Tweet shows a protestor bleeding from the head after being shot by police. The poster claims to have been watching a livestream, in which protestors say the man was struck by a rubber bullet.
```

Comment:
should we include the handle of the poster?

Reply:
done